UnitystationLauncher/UnitystationLauncher.csproj

```diff
@@ -54,8 +54,11 @@
     <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics" Version="0.10.18" />
     <PackageReference Include="Avalonia.ReactiveUI" Version="0.10.18" />
     <PackageReference Include="MessageBox.Avalonia" Version="2.1.0" />
+    <PackageReference Include="Microsoft.ILVerification" Version="8.0.0-preview.6.23329.7" />
     <PackageReference Include="morelinq" Version="3.3.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
With .net6 and newer there really isn't any reason to use newtonsoft json. Its just another nuget package that we have to remember to update when a vulnerability is found. The built in `System.Text.Json` library is very good and is used everywhere else in the launcher that deals with json.